assets/js/scenes/department/ExpenditureSection/index.html

```diff
@@ -52,7 +52,7 @@ <h3 class="Title Title--section">Planned compared to historical expenditure</h3>
                 {% include 'components/LinksList/item.html' with prefix="Source" text=source_text url=dataset type="dataset" %}
 
                 {% include 'components/LinksList/item.html' with text="Download this data for analysis (CSV)" url=csv type="download" %}
-
+                <li> Mahlatse was hereee </li>
```

Comment:
let's remove for the merge :)

Reply:
no changes in this template - it corresponds to a different dataset

Reply:
changes done on the template